frontend/src/components/custom-tools/add-llm-profile/AddLlmProfile.jsx

```diff
@@ -370,7 +373,7 @@ function AddLlmProfile({
         setAlertDetails(
           handleException(
             err,
-            "Failed to get the dropdown list for LLM Adaptors"
+            "Failed to get the chunk size info for LLM Adaptor. Please proceed with default."
```

Comment:
```suggestion
            "Failed to get chunk size information for the requested LLM. Please proceed with a sane default."
```
I don't have full context on this but since this would be user facing, it'll be good to not use any short forms in our message. Also I feel like an adapter is an internal entity and in the UI we don't refer to it as `LLM Adapter` - its just `LLM` so let's stick to that. 

PS: There's a typo as well - its 'Adapter` and not `Adaptor`

Reply:
@chandrasekharan-zipstack Thanks for the updated message. Committing this.